storagenode/storagenodedb/infodb.go

```diff
@@ -23,7 +25,20 @@ var ErrInfo = errs.Class("infodb")
 
 // InfoDB implements information database for piecestore.
 type InfoDB struct {
-	db *sql.DB
+	db        *sql.DB
+	space     *space
+	bandwidth *bw
```

Comment:
these don't need to be pointers

Reply:
also these probably better belong to the pieceinfo implementation rather than the whole infodb

Reply:
updated